app/Http/routes.php

```diff
@@ -40,10 +40,10 @@
     ]);
 
     // Waiting rooms routes.
+    Route::model('waitingrooms', 'Gladiator\Models\WaitingRoom');
```

Comment:
this was just a little OCD kicking in, it doesn't seem to make a difference functionally, but like it seemed out of place stuck in the middle of everything. All it is is defining the model to use with routes names 'waitingrooms' so you can do the type-hinting thing. Seemed better to get that definition out of the way first, and then deal with the route you want.